Packs/OpenLDAP/Integrations/OpenLDAP/OpenLDAP.py

```diff
@@ -30,6 +30,15 @@ class LdapClient:
     SUPPORTED_BUILD_NUMBER = 57352  # required server build number
     CIPHERS_STRING = '@SECLEVEL=1:ECDHE+AESGCM:ECDHE+CHACHA20:DHE+AESGCM:DHE+CHACHA20:ECDH+AESGCM:DH+AESGCM:' \
                      'ECDH+AES:DH+AES:RSA+ANESGCM:RSA+AES:!aNULL:!eNULL:!MD5:!DSS'  # Allowed ciphers for SSL/TLS
+    SSL_VERSIONS = {
+        'SSLv23': ssl.PROTOCOL_SSLv23,
+        'TLSv1': ssl.PROTOCOL_TLSv1,
+        'TLSv1_1': ssl.PROTOCOL_TLSv1_1,
+        'TLSv1_2': ssl.PROTOCOL_TLSv1_2,
+        'TLS': ssl.PROTOCOL_TLS,
+        'TLS_CLIENT': ssl.PROTOCOL_TLS_CLIENT,
+        'TLS_SERVER': ssl.PROTOCOL_TLS_SERVER
```

Comment:
We don't need this one. It is for SERVER only.

Reply:
Removed it. Thanks.